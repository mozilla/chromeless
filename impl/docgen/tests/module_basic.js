/**
 *
 * This is a basic module, it doens't explicitly specify it's own name
 * and it uses the short form for doc-wide comments.
 *
 *   // this is code, it uses some markdown!
 *
 */
