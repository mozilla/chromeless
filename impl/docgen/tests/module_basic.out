{
  "desc": "This is a basic module, it doens't explicitly specify it's own name\nand it uses the short form for doc-wide comments.\n\n  // this is code, it uses some markdown!", 
  "filename": "tests/module_basic.js", 
  "module": "module_basic"
}
